docs/technical-details/tag-filtering.md

```diff
@@ -0,0 +1,158 @@
+# Tag Filtering
+
+During the graph build process, openrouteservice looks at various tags that are given to OSM ways as a means of identifying whether that way should be included into the routing graph or not. For example, in the foot profile any ways that are marked as `highway=motorway` are rejected and not included in the graph, meaning that they can never be routed over (for that profile).
+
+The following tables list what tags are taken into account during the initial filtering process.
+`Reject` means that the tag value indicates that the way is explicitly rejected form the graph building.
+`Accept` means that the tag indicates that the way should be included.
+`Conditional` means that the tag is taken into account during the filtering process,
+but the acceptance/rejection is based on other tags and logic, such as the [`:conditional` suffix](https://wiki.openstreetmap.org/wiki/Conditional_restrictions).
+
+**All ways are initially rejected from the graph building unless criteria for acceptance has been met!**
```

Comment:
This is somehow not true. @aoles wanted to write a few sentences on how graphhopper handles this.

Reply:
I've added a paragraph on how time-dependent restrictions are being currently handled.